tokens/staking/stake.md

```diff
@@ -119,102 +99,161 @@ First, you need to get the `candidateInfo`, which will contain the delegator cou
+You can follow the same steps as described to delegate other candidates in the network.
+
+### Verify Auto-Compounding Percentage {: #verifying-auto-compounding-percentage }
+
+If you want to verify the percentage of rewards that are set to auto-compound for a specific delegation, you can use the following script that will query the `autoCompoundingDelegations` extrinsic and filter the results based on the delegator's address:
+
+```js
+// Simple script to verify your auto-compounding percentage for a given candidate.
+// Remember to replace CANDIDATE-ADDRESS-HERE with the candidate's address you want to delegate
+// And replace DELEGATOR-ADDRESS-HERE with the address used to delegate with
+const candidateAccount = 'CANDIDATE-ADDRESS-HERE'; 
+const delegationAccount = 'DELEGATOR-ADDRESS-HERE'
+const autoCompoundingDelegations = await api.query.parachainStaking.autoCompoundingDelegations(candidateAccount);
+const delegation = autoCompoundingDelegations.find(del => del.delegator == delegationAccount)
+
+console.log(`${delegation.value}%`);
+```
+
+In Polkadot.js Apps, you can head to the **Developer** tab and select **JavaScript** from the dropdown. Then you can take the following steps:
+
+ 1. Copy the code from the previous snippet and paste it inside the code editor box 
+ 2. (Optional) Click the save icon and set a name for the code snippet, for example, **Get auto-compounding percentage**. This will save the code snippet locally
+ 3. To execute the code, click on the run button
+ 4. The result is returned in the terminal on the right side
+
+![Verify auto-compounding percentage](/images/tokens/staking/stake/new/stake-8.png)
+
+## Set or Change the Auto-Compounding Percentage {: #set-or-change-auto-compounding }
+
+If you initially set up your delegation without auto-compounding or if you want to update the percentage on an existing delegation with auto-compounding set up, you can use the `setAutoCompound` function of the Solidity interface.
+
+You'll need to [get the number of delegations with auto-compounding set up](#get-candidate-auto-compounding-count) for the candidate you want to set or update auto-compounding for. You'll also need to [retrieve your own delegation count](#get-your-number-of-existing-delegations). Once you have the necessary information, you can click on the **Developer** tab, select **Extrinsics** from the dropdown, and take the following steps:
+
+ 1. Select the account from which you initially delegated from and want to set or update auto-compounding for
+ 2. Choose the **parachainStaking** pallet
+ 3. Choose the **setAutoCompound** extrinsic
+ 4. Set the candidate's address that you delegated. For this example, it is set to `{{ networks.moonbase.staking.candidates.address1 }}`
+ 5. Set the percentage of rewards to auto-compound by entering a number 0-100
+ 7. For the **candidateAutoCompoundingDelegationHint** field, enter the candidate's number of delegations with auto-compounding configured
```

Comment:
The index numbers are incorrect for these 3 lines. Instead of 7,8,9, they should be 6,7,8.